crt_portal/static/sass/custom/intake.scss

```diff
@@ -95,6 +98,19 @@
     flex-wrap: wrap;
     white-space: normal;
 
+    ul {
+      @include flex-container();
+      flex-wrap: wrap;
+      white-space: normal;
+  
+      li {
+        margin: 0.5rem;
+      }
+      + .usa-button {
+        margin-left: 2rem;
+      }
+    }
```

Comment:
Did we inadvertently change something here? The CSS looks off now:

Plus the dropdown assignee doesn't work for me now. 

![2020-08-03_09-17](https://user-images.githubusercontent.com/3013175/89204081-6373bb00-d56a-11ea-9613-6e168752f445.png)

Reply:
@hakhalid11 thank you for fixing the assignee dropdown! 

The CSS still looks a bit off to me. Here's what it looks like in `develop`:

![2020-08-03_12-13](https://user-images.githubusercontent.com/3013175/89218402-cf621d80-d582-11ea-8da7-882a8f583ee1.png)